src/include/drbd/helps.rb

```diff
@@ -133,6 +133,9 @@ def initialize_drbd_helps(include_target)
         "global_conf"   => _(
           "<p><b><big>Global Configuration of DRBD</big></b></p>"
         ) +
+          _(
+            "<p>Check <b>\"Udev Always Use VNR\"</b> treat implicit the same as explicit volumes, only one single volume with the implicit volume number 0</p>"
```

Comment:
Maybe "Check .... to treat the implicit volumes the same way as the explicit ones". I'm not sure what the trailing note about the volume number 0 means, could you describe it better, please?

Reply:
drbd resource support multiple volumes. In this case, user may explicit define the volume number for each disk.
I will add more in the help